data/presets/fields/surveillance.json

```diff
@@ -0,0 +1,8 @@
+{
+    "key": "surveillance",
+    "type": "combo",
+    "label": "Surveillance kind",
+    "default": "outdoor",
+    "placeholder": "Public, Outdoor, Indoor",
```

Comment:
No need to add placeholder here, this will be computed automatically from available values.. Don't add Uppercase placeholders for a lowercase tag..